packages/sdk-react/src/components/approvalPolicies/ApprovalPolicyDetails/ApprovalPolicyView/utils/getVariantProps.ts

```
@@ -0,0 +1,139 @@
+        showAvatar: false,
+        showIcon: false,
+      };
+    case 'outlined':
+      return {
+        useBadge: true,
+        badgeVariant: 'outline',
+        className: `mtw:inline-flex mtw:items-center mtw:justify-center mtw:text-sm mtw:font-medium mtw:whitespace-nowrap mtw:rounded mtw:px-2 mtw:py-1 mtw:gap-1 ${className}`,
+        showAvatar: false,
+        showIcon: true,
+      };
+    case 'default':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)